README.md

```diff
@@ -20,6 +20,10 @@ To run:
   - Network problems may fail the process. First try running the command again. Intermediate states are saved so not everything will be redone even when there is a failure.
 5. Provide an environment file (.env) in the root folder with the following keys
 ```
+# Windows only - path to the root directory of the project with forward slashes
+# such as /c/User/me/biab-2.0
```

Comment:
Never seen that syntax. I trust you that it works ?

Reply:
This syntax is the way linux emulators usually access drives on a windows host. You will see that when you navigate with git bash for instance.
I seems that this is what docker compose uses as well! I don't remember where I saw that, since I had to search the web far and wide...